CoreScriptsRoot/Modules/PlayerDropDown.lua

```diff
@@ -215,8 +215,15 @@ local function GetBlockedPlayersAsync()
 			local request = HttpRbxApiService:GetAsync(apiPath)
 			blockList = request and game:GetService('HttpService'):JSONDecode(request)
 		end)
-		if blockList and blockList['success'] == true and blockList['userList'] then
-			return blockList['userList']
+		if success then
```

Comment:
This was some leftover code from debugging that I forgot to remove. I was initially using that to check if the issue was coming from the web request.